packages/lib-react-components/src/Modal/Modal.stories.js

```diff
@@ -73,3 +75,129 @@ export function Required({
     </Modal>
   )
 }
+
+// Modal styling for eventual SubjectGroupViewer Done&Talk modal starts here
+const StyledAnchor = styled(Anchor)`
+  &:hover, &:focus, &:focus-visible {
+    border: solid 3px ${props => props.theme.global.colors.brand};
+  }
+`
+
+export function SubjectGroupTalkExample() {
+  return (
+    <Modal
+      active={true}
+      closeFn={() => true} // closeFN must be present to close the modal and show the X close button
+      title='Discuss this subject group'
+    >
+      <Box gap='xsmall' width={{ max: '400px' }}>
+        <Text>Choose a subject to view in Talk</Text>
```

Comment:
Note that in the real classifier, these strings will need to be new translation keys.